spec/support/fake_attempts_tracker.rb

```diff
@@ -0,0 +1,50 @@
+module IrsAttemptsApiTrackingHelper
+  class FakeAttemptsTracker
+
+    include IrsAttemptsApi::TrackerEvents
+
+    attr_reader :events
+
+    def initialize
+      @events = Hash.new
+    end
+
+    def track_event(event, attributes = {})
+      events[event] ||= []
+      events[event] << attributes
+      nil
+    end
+
+    def track_mfa_submit_event(_attributes)
+      # no-op
+    end
+
+    def browser_attributes
+      {}
+    end
+  end
+
+  RSpec::Matchers.define :have_logged_event do |event_name, attributes|
```

Comment:
so I think matchers are defined globally... since this interface of this is basically the same as the FakeAnalytics one... we shouldn't redefine this.

Also, this is a legacy of when we did string event names, but now that we're defining and stubbing specific methods, I think we can get rid of it here

Reply:
Great - code removed and the test still pass! 😄